### PR TITLE
Add NUMERIC Type

### PR DESCRIPTION
**Pull request by HokutoMorita**
# What I did
Implementation of conversion process from String to NUMERIC

## ToDo
  - [x] Specification confirmation to @hiro-koba 

## Issue
https://github.com/primenumber-dev/n-transfer-ui/issues/9864

### CODE REVIEW

**Review by kekekenta — APPROVED** [review]
LGTM

**Comment by HokutoMorita** [comment]
@d-hrs @giwa 
レビューよろしくお願いいたします！

File: src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java [hunk 1]
```diff
@@ -106,6 +108,11 @@ public static void convertAndSet(ObjectNode node, String name, String src, Bigqu
                     }
                 }
                 break;
+            case NUMERIC:
+                // BigQuery NUMERIC型のスケール最大値が9なのでデフォルト値を9にする
```
Comment (d-hrs) [inline src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java:112]: English please

File: build.gradle [hunk 2]
```diff
@@ -12,7 +12,7 @@ repositories {
 }
 
 group = "io.trocco"
-version = "0.0.21"
+version = "0.0.22"
```
Comment (d-hrs) [inline build.gradle:15]:
```suggestion
version = "0.1.0"
```

**Review by d-hrs — CHANGES_REQUESTED** [review]
In this repository, please do everything in English.

File: src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java (hunk 1 above)
Comment (HokutoMorita) [reply src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java:112]: got it !

Comment (HokutoMorita) [reply src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java]: Fixed https://github.com/trocco-io/embulk-output-bigquery_java/pull/93/commits/237377bc04f71c1f6edb83ebd639555d8b2ba895

File: build.gradle (hunk 2 above)
Comment (HokutoMorita) [reply build.gradle]: Fixed https://github.com/trocco-io/embulk-output-bigquery_java/pull/93/commits/68bc5aad2d0c0dc2f0550a7a34ead1ba34eae2c5

**Comment by HokutoMorita** [comment]
@d-hrs 
Please code review

**Comment by d-hrs** [comment]
please change title & description in English.

**Comment by HokutoMorita** [comment]
@d-hrs 
Please code review

**Comment by d-hrs** [comment]
> [Java版][ADD]NUMERIC対応 #93

in English please.

**Comment by HokutoMorita** [comment]
> > [Java版][ADD]NUMERIC対応 #93
> 
> in English please.

@d-hrs 
Thank you for making sure
I fixed

**Comment by HokutoMorita** [comment]
@d-hrs
Please review

**Review by d-hrs — APPROVED** [review]

**Pull request merged**